auth-web/src/components/auth/manage-business/AddNameRequestForm.vue

```diff
@@ -4,32 +4,29 @@
       <fieldset>
         <legend hidden>Name Request Number and Applicant Phone Number or Email Address</legend>
         <v-text-field
-          filled
+          ref="nrNumber"
+          filled persistent-hint validate-on-blur
```

Comment:
`req` is mutually exclusive with validate-on-blur (and checked via a rule anyway).

`validate-on-blur` is used to perform rule validation when the user leaves this component.